publication-commons/src/main/java/no/unit/nva/publication/model/business/PublishingRequestCase.java

```diff
@@ -341,9 +341,7 @@ && getStatus() == that.getStatus()
     public PublishingRequestCase persistAutoComplete(TicketService ticketService, Publication publication,
                                                      Username finalizedBy)
         throws ApiGatewayException {
-        var ticket = this.persistNewTicket(ticketService);
-        ticket.complete(publication, finalizedBy).persistUpdate(ticketService);
-        return (PublishingRequestCase) ticket.fetch(ticketService);
```

Comment:
We do not need this fetch, cause `persistNewTicket()` fetches ticket after insertion:
```
    private <T extends TicketEntry> T createTicketForPublication(Publication publication,
                                                                 TicketEntry ticketEntry)
        throws ConflictException {

        setServiceControlledFields(ticketEntry, identifierProvider);
        ticketEntry.validateCreationRequirements(publication);
        var request = ticketEntry.toDao().createInsertionTransactionRequest();
        sendTransactionWriteRequest(request);
        FunctionWithException<TicketEntry, TicketEntry, NotFoundException>
            fetchTicketProvider = this::fetchTicket;
        return (T) fetchEventualConsistentDataEntry(ticketEntry, fetchTicketProvider).orElseThrow();
    }
```